rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.get/10_basic.yml

```diff
@@ -169,3 +169,15 @@ setup:
      catch: bad_request
      indices.get:
        index: _foo
+
+---
+"Should return only selected features":
+
```

Comment:
I don't fully understand why skip version section is not needed here.
When this yaml test is ran against 7.17 then a 7.17 node will fail here, 
because it doesn't know the `features` query string parameter?

Reply:
Hmm, note that this test did pass in the BWC test suite below with a 7.17.0 node, though it is quite possible that I misunderstood how it passed.

Reply:
Maybe the test hit a 8.1.0-SNAPSHOT node during this test run and not a 7.17.0 node?
I would expect this test to fail in a mixed version cluster. Maybe not every time, but at least sometimes.

Reply:
Maybe re-run bwc job with `test-full-bwc` label attached to this pr?

Reply:
Thanks, @martijnvg. That explains it. I'll add the skip version to the test to fix that.